lib/phase/structural.js

```diff
@@ -3,6 +3,37 @@
 const Joi = require('joi');
 const SCHEMA_CONFIG = require('screwdriver-data-schema').config.base.config;
 
+/**
+ * Check if user teardown is at the end of the yaml
+ * @method checkUserTeardown
+ * @param  {Object}          data Screwdriver yaml
+ * @return {Promise}              Error if there is an error
+ */
+function checkUserTeardown(data) {
+    let steps = [];
+    let firstUserTeardownStep = -1;
+    let error = null;
+
+    Object.keys(data.jobs).forEach((job) => {
+        steps = data.jobs[job].steps;
+
+        if (steps) { // if there are user-defined steps
+            for (let i = 0; i < steps.length; i += 1) {
+                const isUserTeardown = Object.keys(steps[i])[0].startsWith('user-teardown-');
```

Comment:
can we just call it `teadown-` ?  Why make it more verbose with `user-` prefix.

Reply:
Sure I can change that